docs/operator-guide/add-ons-overview.md

```diff
@@ -67,33 +67,41 @@ To enable Add-Ons, it is necessary to have the configured Argo CD, and connect a
     * Cluster URL - select `URL` or cluster Name;
     * Namespace - enter the namespace which must be equal to the **Project name** field;
 
-8. As soon as the repository is synchronized, the list of applications that can be installed by Add-Ons will be shown:
+8. Click the **Sync** button to see the list of applications that can be installed by Add-Ons:
 
     ![Add-Ons list](../assets/operator-guide/add_ons_to_install.png "Add-Ons list")
 
 ## Install Add-Ons
 
 Now that Add-Ons are enabled in Argo CD, they can be installed by following the steps below:
 
-1. Choose the Add-On to install.
+1. Decide which Add-Ons you want to install. All the Add-Ons are listed in the [/clusters/core/apps/templates](https://github.com/epam/edp-cluster-add-ons/tree/main/clusters/core/apps/templates) file.
 
-2. On the chosen Add-On, click the **⋮** button and then **Details**:
+2. Enable the Add-Ons chosen in the [/clusters/core/apps/values.yaml](https://github.com/epam/edp-cluster-add-ons/blob/main/clusters/core/apps/values.yaml) file. You can also re-define Add-On parameters. Refer to the example below:
```

Comment:
You can also re-define Add-On parameters. Refer to the example below:

You cant redefine addon parameter
In this sections you could only enable or disable it

Reply:
In this file